_security/multi-tenancy/multi-tenancy-config.md

```diff
@@ -15,19 +15,23 @@ config:
   dynamic:
     kibana:
       multitenancy_enabled: true
+      private_tenant_enabled: true
+      default_tenant: global tenant
       server_username: kibanaserver
       index: '.kibana'
     do_not_fail_on_forbidden: false
 ```
 
-Setting | Description
-:--- | :---
-`multitenancy_enabled` | Enable or disable multi-tenancy. Default is true.
-`server_username` | Must match the name of the OpenSearch Dashboards server user from `opensearch_dashboards.yml`. Default is `kibanaserver`.
-`index` | Must match the name of the OpenSearch Dashboards index from `opensearch_dashboards.yml`. Default is `.kibana`.
-`do_not_fail_on_forbidden` | If true, the security plugin removes any content that a user is not allowed to see from search results. If false, the plugin returns a security exception. Default is false.
+| Setting | Description |
+| :--- | :--- |
+| `multitenancy_enabled` | Enable or disable multi-tenancy. Default is true. |
```

Comment:
Global: Put all of these default settings, in both tables, in code format, as I've done here.